src/alert-manager/deploy/alert-manager-deployment.yaml.template

```diff
@@ -53,22 +52,45 @@ spec:
         volumeMounts:
         - name: config-volume
           mountPath: /etc/alertmanager
-        - name: templates-volume
-          mountPath: /etc/alertmanager/template
         - name: alertmanager
           mountPath: /alertmanager
+{% if cluster_cfg["alert-manager"]["alert-handler"]["configured"] %}
```

Comment:
> Seems we can remove `configured` flag since it always enabled to show alerts in webportal

Alerts in web portal is independent of `alert-handler`. So `configured` will only be true when at least one of the other actions is available.